api/openapi.gen.json

```diff
@@ -286,6 +250,42 @@
           }
         }
       }
+    },
+    "/sources/{ID}/instance_types": {
```

Comment:
@akhil-jha @mshriver we're renaming the endpoint, does this makes more sense? Will that be easier to test?